doc/payum-example/src/AppBundle/Payment/StripeV3RequirementsFulfillerOnCaptureExtensions.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * Created by Bruno DA SILVA, working for Combodo
+ * Date: 08/08/18
+ * Time: 15:12
+ */
+namespace AppBundle\Payment;
+
+
+use Payum\Core\Extension\Context;
+use Payum\Core\Extension\ExtensionInterface;
+use Payum\Core\Request\Capture;
+use Payum\Core\Request\GetHumanStatus;
+use Payum\Core\Model\Payment;
+
+/**
+ *
+ * This class handles the addition of line_items to stripe checkout v3
+ *
+ * It is wired using Payum's built in extension system
+ *  - @see https://github.com/Payum/PayumBundle/blob/master/Resources/doc/container_tags.md#extension-tag
+ *  - @see https://github.com/Payum/Payum/blob/master/docs/the-architecture.md#extensions
+ *
+ * It is wired to stripe_checkout only using a factory filter on the service tag.
+ * It is additionally filtered to the Capture request using a code check.
+ *
+ * for the explanation of why line_items are required, please refer to
+ *  - https://stripe.com/docs/payments/checkout/server#integrate
+ *  - https://stripe.com/docs/api/checkout/sessions/create#create_checkout_session-line_items
+ *
+ * @author Bruno DA SILVA
+ * @author Valentin Corre
+ */
+class StripeV3RequirementsFulfillerOnCaptureExtensions implements ExtensionInterface
```

Comment:
Maybe the classes should be named differently than the ones in the sylius example?
Or it may be mentioned that they are the same?

Reply:
Is StripeV3PreparePaymentExtension & StripeV3ProcessPaymentExtension ok for you ?

Reply:
:+1: They are so perfect that I jealous! :)